Galaxus.Functional.NUnitExtension/Is.cs

```diff
@@ -0,0 +1,9 @@
+namespace Galaxus.Functional.NUnitExtension;
+
+public class Is : NUnit.Framework.Is
+{
+    public static NoneConstraint OptionOfNone => new NoneConstraint();
```

Comment:
Instead of `Is.OptionOfSome` or `Is.OptionOfNone`, what about simply using `Is.Some` or `Is.None`? While `Some` and `None` do exist, they are not starters, so I'd prefer the name-clash over redundant "is an option". What do you think?

Reply:
Even more than that, I just asked myself: Do we really need to go `Is.Some.WithValue(x)` - what about going directly to `Is.Some(x)`? Same for all the others, of course.

Reply:
In my first iteration I had exactly the `Is.Some` and `Is.None` pattern. I then thought that, especially the wording `None` is not excluslivly bound to Options. Therefore I went with `OptionOfSome` and `OptionOfNone` to prevent confusion and possible future naming collisions with NUnit itself.

But I am open for naming suggestions. So if you prefer, we can go with Some and None